src/edu/ucsb/nceas/metacat/dataone/v1/MNodeService.java

```diff
@@ -83,7 +83,7 @@
 	edu.ucsb.nceas.metacat.dataone.MNodeService impl = null;
 	
 	/* the logger instance */
-    private Logger logMetacat = null;
+    private org.apache.commons.logging.Log logMetacat = null;
```

Comment:
```suggestion
    private org.apache.commons.logging.Log logMetacat = LogFactory.getLog(MNodeService.class);
```

Reply:
See the commit 70db73cab0ce506277782cd20f081b0dabcd9c8a.